### DISCUSSION TITLE
no playback on youtube videos?

### DISCUSSION THREAD

**Discussion by arqueiro** [Yisus7u7/termux-desktop-xfce#61] [General]
I am getting an error when trying to play videos on youtube.  Anyone else? "An error occurred, please try again later"

**Comment by JesusChapman** [comment]
![Screenshot_20211129-200915](https://user-images.githubusercontent.com/64093255/144249030-ac662d63-5a50-45a1-8ba7-565375d47f54.png)

It is a bug that I am working on fixing, a package called glib-network must be added to termux for the browser to play videos.
